tests/test_cli_contract_help.py

```
@@ -0,0 +1,213 @@
+    except Exception as exc:
+        print(f"FAIL: {exc}")
+        return 1
+
+    failures: list[str] = []
+    for probe in probes:
+        try:
+            result = run_probe(cli_path, probe)
+        except subprocess.TimeoutExpired:
+            failures.append(f"{probe.command}: timed out")
+            continue
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)